bahmni-web/resources/httpd.conf

```diff
@@ -587,8 +526,7 @@ ScriptAlias /cgi-bin/ "/var/www/cgi-bin/"
 <Directory "/var/www/cgi-bin">
     AllowOverride None
     Options None
-    Order allow,deny
-    Allow from all
+    Require all granted
```

Comment:
@shilpa-iplit Why we need all granted?

Reply:
@SwathiVarkala Require all granted is for authorize users I guess